docs/src/multi-physics_coupling.md

```diff
@@ -0,0 +1,37 @@
+# [Multi-physics coupling](@id multi-physics-coupling)
+A complex simulation can consist of different spatial domains in which
+different equations are being solved, different numerical methods being used
+or the grid structure is different.
+One example would be a fluid in a tank and an extended hot plate attached to it.
+We would then like to solve the Navier-Stokes equations in the fluid domain
+and the heat conduction equations in the plate.
+The coupling would happen at the interface through the exchange of thermal energy.
+
+
+## Converter Coupling
+We can have the case where the two systems do not share any variables, but
+share some of the physics.
+Here, the same physics is just represented in a different form and with
+different variables.
+This is the case for a fluid system on one side and a Vlasov system on the other.
+To translate the fields from one description to the other one needs to use
+converter functions.
+
+In the general case, we have a system A with ``m`` variables ``u_{A,i}`` and another
+system B with ``n`` variables ``u_{B,j}``.
```

Comment:
A bit of bikeshedding: I just saw that this is already merged, but as far as I can tell, this should actually be with the double backticks, see [documenter docs](https://documenter.juliadocs.org/stable/man/latex/#Inline-Equations) and https://github.com/trixi-framework/Trixi.jl/pull/1558#discussion_r1370655579. I just checked that we use both version across the docs. I can create a PR that unifies this to use the two backticks version. Or was not any other reason for the suggestion, which I don't see, @DanielDoehring?

Reply:
It you could do this, it would be very nice, thanks

Reply:
But let's wait for Daniel's input

Reply:
The reason I changed this is because (at least as far as I know) documenter can handle both double back ticks and dollar signs for displaying math, but standard markdown does only accept dollar signs, see

$m$ vs ``m``.

So the version with dollar signs should be nicely displayed outside of the doc only, while the version with double back ticks is essentially displayed as code:

https://github.com/trixi-framework/Trixi.jl/blob/09940118196206710d8b3e6ef9abba405701d0f5/docs/src/multi-physics_coupling.md#converter-coupling

Reply:
Makes sense. However, it is still not the way how Documenter.jl recommends using LaTeX syntax. I found the [discussion about that in Documenter.jl](https://github.com/JuliaDocs/Documenter.jl/pull/1718#discussion_r733474944).

Reply:
Maybe discuss this at Trixi meeting?